src/index.d.ts

```diff
@@ -0,0 +1,65 @@
+import * as resultTypes from "./types";
+
+declare namespace PlantUMLParser {
+  export class Class {
```

Comment:
In order to avoid maintenance of the classes in two places, I have change the project so that the parsing expression grammar uses classes generated from typescript classes (in `dist/`). As an example I have converted `Note` to typescript. The idea is that all those clases which are currently stored in single files can now be moved to `types.ts` and should be removed from here.

Reply:
So just to be clear, would you like me to move all exported classes into `types.ts` and delete those files?

Reply:
yes. Thats 100% correct. All those classes should be moved to `types.ts` and the `.js` files should be removed.

Reply:
Ok, noted.